resources/sql/delete_orgunittree_with_data.sql

```diff
@@ -81,6 +81,20 @@ EXECUTE 'update reporttable_organisationunits set sort_order = -(sort_order+1) w
 END LOOP;
 EXECUTE 'TRUNCATE temp1';
 
+
+--visualization_organisationunits
+EXECUTE 'DELETE FROM visualization_organisationunits WHERE organisationunitid = $1 ' USING organisationunitid;
+EXECUTE 'INSERT INTO temp1 SELECT visualizationid from visualization_organisationunits where organisationunitid = $1 'USING organisationunitid;
```

Comment:
I presume this works, as the delete in row 86 is not yet committed - but does it have an advantage over inserting into the temp1 table first, then deleting? I would even consider completing the loop, and then deleting after everything has gone well.

Reply:
Hi. Using the holidays to clean up PRs finally.
I only added a code to delete visualizations. For that I copy pasted code that was already worked out by someone else. I know the code works because I test it but I don't have time for such an extensive review. There are probably more efficient ways to delete but for the moment I am just happy it works